### PR TITLE
Feat(ui): Add secrets admin

### PR DESCRIPTION
**Pull request by uwe-mayer**
Spin up your local setup as described in the [README](https://github.com/cloudoperators/greenhouse/compare/feat/ui/secret-admin-view?expand=1#diff-9bd83305b4bec8da6217246d88ae426dccecd08dc7412a078e106d61b7dd8fc0) to give this admin view a spin...

### CODE REVIEW

**Comment by uwe-mayer** [comment]
@ArtieReus @TilmanHaupt 
If you could have a look, since this is the refactoring extracting secrets from https://github.com/cloudoperators/greenhouse/pull/142

File: ui/secret-admin/package.json [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      "type": "optional",
+      "description": "mock the OIDC data, allowed values are 'true', 'false' (default), or json (pure or base64 encoded)"
+    },
+    "theme": {
+      "value": "theme-dark",
+      "description": "Override the default theme. Possible values are theme-light or theme-dark (default)"
+    },
+    "embedded": {
+      "value": "false",
+      "description": "Set to true if app is to be embedded in another existing app or page, like e.g. Elektra.  If set to true the app won't render a page header/footer and instead render only the content"
```
Comment (ArtieReus) [inline ui/secret-admin/package.json:89]: We probably need to remove the example from Elektra. It is not related in this context.

**Review by ArtieReus — CHANGES_REQUESTED** [review]

File: ui/secret-admin/package.json [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    "@testing-library/user-event": "^14.4.3",
+    "assert": "^2.0.0",
+    "autoprefixer": "^10.4.2",
+    "babel-jest": "^29.3.1",
+    "babel-plugin-transform-import-meta": "^2.2.0",
+    "communicator": "https://assets.juno.global.cloud.sap/libs/communicator@2.2.6/package.tgz",
+    "esbuild": "^0.19.12",
+    "jest": "^29.3.1",
+    "jest-environment-jsdom": "^29.3.1",
+    "juno-ui-components": "https://assets.juno.global.cloud.sap/libs/juno-ui-components@2.13.5/package.tgz",
```
[the remark this answers is not in the document]
Comment (uwe-mayer) [reply ui/secret-admin/package.json]: :heavy_check_mark:

File: ui/secret-admin/package.json [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    "sapcc-k8sclient": "^1.0.2",
+    "sass": "^1.60.0",
+    "shadow-dom-testing-library": "^1.7.1",
+    "tailwindcss": "^3.3.1",
+    "url-state-provider": "https://assets.juno.global.cloud.sap/libs/url-state-provider@1.3.2/package.tgz",
+    "util": "^0.12.4",
+    "zustand": "4.3.7"
+  },
+  "peerDependencies": {
+    "juno-ui-components": "*",
```
[the remark this answers is not in the document]
Comment (uwe-mayer) [reply ui/secret-admin/package.json]: removed from the peer dependencies altogether

File: ui/secret-admin/package.json [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    "luxon": "^2.3.0",
+    "prop-types": "^15.8.1",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "url-state-provider": "*",
```
[the remark this answers is not in the document]
Comment (uwe-mayer) [reply ui/secret-admin/package.json]: removed from the peer dependencies altogether

File: ui/secret-admin/src/App.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  const queryClient = new QueryClient({
+    defaultOptions: {
+      queries: {
+        meta: {
+          endpoint: props.endpoint || "",
+        },
+      },
+    },
+  })
+  useCommunication()
```
[the remark this answers is not in the document]
Comment (uwe-mayer) [reply ui/secret-admin/src/App.js]: removed from App.js

**Review by ArtieReus — APPROVED** [review]
lgtm

**Pull request merged**